const test = 1
let b = test / 0
throw err
